app/code/Magento/Customer/Model/AccountManagement.php

```diff
@@ -1159,8 +1165,8 @@ public function isResetPasswordLinkTokenExpired($rpToken, $rpTokenCreatedAt)
 
         $expirationPeriod = $this->customerModel->getResetPasswordLinkExpirationPeriod();
 
-        $currentTimestamp = (new \DateTime())->getTimestamp();
-        $tokenTimestamp = (new \DateTime($rpTokenCreatedAt))->getTimestamp();
+        $currentTimestamp = ($this->dateTimeFactory->create())->getTimestamp();
+        $tokenTimestamp = ($this->dateTimeFactory->create($rpTokenCreatedAt))->getTimestamp();
```

Comment:
Too fast :) 2.1 still supports PHP5.6:
```
PHP Parse error:  syntax error, unexpected '->' (T_OBJECT_OPERATOR) in /home/travis/build/magento/magento2/app/code/Magento/Customer/Model/AccountManagement.php on line 1168
```

Reply:
Ouch, I forgot remove wrapping parenthesis in this one, I've reviewed also 2.2 and 2.3 PRs and they are ok. Updated.

Reply:
I've been also playing with PhpStorm to configure code reformatting, static tests are still complaining about that, I'm fixing it.